posthog/migrations/0356_alter_messagingrecord_unique_together.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.19 on 2023-10-24 02:53
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    atomic = False  # dropping constraint can't be done atomically
+    dependencies = [
+        ("posthog", "0355_add_batch_export_backfill_model"),
+    ]
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            state_operations=[
+                migrations.AlterUniqueTogether(
+                    name="messagingrecord",
+                    unique_together={("email_hash", "campaign_key", "campaign_count")},
+                ),
+            ],
+        ),
+        migrations.RunSQL(
+            sql="""
+            ALTER TABLE posthog_messagingrecord DROP CONSTRAINT "posthog_messagingrecord_email_hash_campaign_key_6639a13f_uniq";
+            """,
+            reverse_sql="""
```

Comment:
i don't think we should have the reverse_sql here, if we need to rollback we should manage it manually (creating this constraint could cause the lockup issues we were trying to avoid by doing it all concurrently)